### DISCUSSION TITLE
Should the comment TODO behavior be changed?

### DISCUSSION THREAD

**Discussion by The-Futurist** [dotnet/roslyn#57898] [Ideas] [answered]
I've been writing TODO comments more recently as reminders to revisit certain areas of code that might require refinement.

I found that if I write something like this:

```cs
// TODO we must do some stuff in the future here because
// we can't be certain in all cases that what we assumed at 
// the outset is still applicable partway through.
```

or

```cs
/*
TODO we must do some stuff in the future here because
we can't be certain in all cases that what we assumed at 
the outset is still applicable partway through.
*/
```

then the task list window only show the comment text between the opening of the TODO comment and the next end of line.

I'd find it more natural if either the entire comment block was printed in the task list or all of the text up to and including the next period.

Admittedly this is a small issue but it would improve the experience for me when I have a lot of these comments, when written in a hurry (as some often are) the user might not be conscious of this end of line behavior and might just not see certain parts of important TODO comments weeks later when scanning the task list for these.

**Comment by CyrusNajmabadi** [comment, marked as answer]
That seems reasonable. Another alternative in the meantime might be too preface each line with TODO.

**Reply by CyrusNajmabadi** [reply]
I've converted this to an issue to track this feature request: https://github.com/dotnet/roslyn/issues/57899
